.vitepress/config.mts

```diff
@@ -61,6 +62,7 @@ const config = defineConfig({
         /\.php/,
         /ocramius/,
     ],
+    appearance: false // This disables the dark mode toggle
```

Comment:
Does this work? I already tested it with no luck, so I had to hide the toggle button manually using CSS.

Reply:
Yep! This made the entire website "white". Since our website uses white theme, lets disable this feature (until we have a fully supported dark theme)